Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -35,13 +35,11 @@ public class Game {
 		view.assignBlanks(word, correctGuesses: guessManager.correctGuesses)
 		let guess = view.receiveGuess()
 		
-		if (guessManager.isGuessingFullWord(guess)) {
-			guessManager.appendWordPlaceholder(word, guess: guess)
-		} else {
-			guessManager.appendGuess(word, guess: guess)
-		}
+		guessManager.appendGuess(word, guess: guess)
+		let incorrectGuesses = guessManager.incorrectGuesses
 
 		view.displayRemainingGuesses(guessManager.calculateRemainingGuesses())
+		view.displayIncorrectGuesses(incorrectGuesses)
 		isGameOver(guess)
```

Comment:
I think this has been updated since this comment. I was trying to separate the functional sections of the method, user interaction, guess logging, user interaction, guess evaluating. I reduced the lines a bunch so I can probably just put everything together.